tools/crates/BUILD.gn

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/tools/crates/cargo.gni")
+import("//brave/tools/crates/config.gni")
+
+action("build_crates") {
+  script = "//build/gn_run_binary.py"
+
+  inputs = [
+    cargo_exe,
+    "Cargo.lock",
+  ]
+
+  outputs = [
+    cargo_audit_exe,
+    wasm_bindgen_cli_exe,
+    wasm_opt_exe,
+    wasm_pack_exe,
+  ]
+
+  args = [
+    rebase_path(cargo_exe),
+    "build",
+    "--release",
+    "--manifest-path",
+    rebase_path("Cargo.toml"),
+    "--config",
+    rebase_path(".cargo/config.toml"),
+    "--target-dir",
+    rebase_path(target_dir),
+    "--frozen",
+    "-p",
+    "cargo-audit",
```

Comment:
consider splitting `cargo-audit` and `wasm`-related crates build into two separate gn targets.

I think it's possible to have a gn template to build crates that can accept a list of rust packages so you can do something like  `foreach (package_name, package_names) { args += ['-p', package_name] }`.

Reply:
We can also build them one by one and trigger the wasm-related crates via `deps`: https://github.com/brave/brave-core/pull/27475/commits/2afea2d02c32621e734318200266b35dc8f2ae83

Reply:
We can also build them one by one and trigger the wasm-related crates via `deps`: https://github.com/brave/brave-core/pull/27475/commits/bca53c519c922ba2da09ceebcf45eb1916bf1772